From: Andrew Morton <akpm@linux-foundation.org>
Date: Fri, 13 Dec 2024 16:35:29 -0800
Subject: [PATCH] 
 mm-fix-outdated-incorrect-code-comments-for-handle_mm_fault-fix

s/mmap_Lock/mmap_lock/, per Liam

Cc: Jinliang Zheng <alexjlzheng@gmail.com>
Cc: Jinliang Zheng <alexjlzheng@tencent.com>
Cc: "Liam R. Howlett" <Liam.Howlett@oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 mm/memory.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/memory.c b/mm/memory.c
index 5ebad355b245..4fe0404fa198 100644
--- a/mm/memory.c
+++ b/mm/memory.c
@@ -6141,7 +6141,7 @@ static vm_fault_t sanitize_fault_flags(struct vm_area_struct *vma,
 
 /*
  * By the time we get here, we already hold either the VMA lock or the
- * mmap_Lock (FAULT_FLAG_VMA_LOCK tells you which).
+ * mmap_lock (FAULT_FLAG_VMA_LOCK tells you which).
  *
  * The mmap_lock may have been released depending on flags and our
  * return value.  See filemap_fault() and __folio_lock_or_retry().
-- 
2.39.5

